engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java

```diff
@@ -21,6 +21,7 @@ private SortableFieldMetadata(String fieldName) {
 
 	/**
 	 * The name of the field whose field bridge will be used to determine the value for the doc value field.
+	 * @return the name of the field
```

Comment:
Thinking about it, I believe that's only reported as WARN by JavaDoc. It's some other actual ERROR reports (such as unpaired tags, stale links) which make it fail. I don't think we need (or even should) fix everything reported at WARN level. That's not to suggest to undo it, just to describe the strategy I'd follow in general.